formation/fields.py

```diff
@@ -2,6 +2,8 @@
 from django.utils.safestring import mark_safe
 from django.utils.translation import ugettext_lazy as _
 from django.core.validators import EmailValidator, URLValidator
+from project.jinja2 import oxford_comma
+from markupsafe import escape
```

Comment:
I don't think this is used anywhere in this file. Maybe a leftover?

Reply:
escape is, but oxford comma is used